drivers/ce_performance/app/views/ce_performance/warehouse_reports/reports/_event_breakdown_table.haml

```diff
@@ -10,9 +10,9 @@
     %table.table.table-striped
       %thead
         %tr
-          %th Event Type
-          %th= comparison_year
-          %th= report_year
+          %th{ width: '80%' } Event Type
+          %th{ width: '20%' }= comparison_year
+          %th{ width: '20%' }= report_year
```

Comment:
This is probably fine, but I think the following is probably a bit cleaner and _should_ do the same thing. (Though please test if you update it.)
```suggestion
          %th.w-80 Event Type
          %th.w-20= comparison_year
          %th.w-20= report_year
```

Reply:
it looks like only w-25/50/75/100 are available as classes. I'll get these updated so the first column is using 75% width class. This column is the one driving the width disparities on this page so that should be enough to fix the alignment.